client/webserver/site/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "dexclient",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "dexclient",
-      "version": "1.1.0",
+      "version": "1.2.0",
       "license": "Blue Oak 1.0.0",
```

Comment:
This thing checked in here too. Reverting...

Reply:
Reverted. https://github.com/decred/dcrdex/compare/7d192d63c92c9483324b634a0d694c0dede19a6c..ce32862ef975be44d1370b19382bc543dbc814bb

This is set to `1.2.0` whenever I run `npm install`

Reply:
> Reverted. https://github.com/decred/dcrdex/compare/7d192d63c92c9483324b634a0d694c0dede19a6c..ce32862ef975be44d1370b19382bc543dbc814bb
> 
> This is set to `1.2.0` whenever I run `npm install`

You should only run `npm clean-install` unless you are trying to update a dependency to something that is beyond what's specified in package-lock.json.